Dockerfile

```diff
@@ -6,7 +6,7 @@
 # Stage 1 - Backend build environment
 # includes compilers and build tooling to create the environment
 
-FROM python:3.7-alpine AS backend-build
+FROM python:3.9-alpine AS backend-build
```

Comment:
Minor detail, but it seems that `COMMIT_HASH` is not used in the Docker image: https://github.com/maykinmedia/zds-token-issuer/runs/7563531224?check_suite_focus=true#step:4:474